src/client/components/UserHeader.js

```diff
@@ -83,7 +84,12 @@ const UserHeader = ({
             )}
           </div>
           <div className="UserHeader__handle-rank-container">
-            <div className="UserHeader__row UserHeader__handle">@{handle}</div>
+            <div className="UserHeader__row UserHeader__handle">
+              @{handle}&nbsp;
+              {!isSameUser && isFollowingYou && (
```

Comment:
I don't really like this triple conditional expression. It makes the code look ugly. Can you put this up top and create a constant like `const displayFollowsYou = !isSameUser && isFollowingYou`

Reply:
his provides way more context on what the expression actually does